.github/scripts/collect-metrics.sh

```
@@ -0,0 +1,225 @@
+# Pipeline Metrics Collection Script
+# Collects and reports CI/CD pipeline performance metrics
+#
+# Usage:
+#   ./collect-metrics.sh [--workflow=WORKFLOW_NAME] [--days=N] [--format=json|table]
+#
+
+set -euo pipefail
+
+# Default values
+WORKFLOW_NAME="${1:-}"
+DAYS="${2:-7}"
```

Shellcheck (0.11.0)

[warning] DAYS appears unused. Verify use (or export if used externally).

(SC2034)